### PR TITLE
helm: Using own namespace

### PR DESCRIPTION
**Pull request by guyarb**

### CODE REVIEW

File: helm/seekret/templates/seekret.yaml [hunk 1]
```diff
@@ -173,7 +180,7 @@ data:
     - name: sniffer
       env:
       - name: S3_URL
-        value: {{ .Values.s3.s3URL | quote }}
+        value: {{ .Values.s3.s3URL }}
```
Comment (asdftamir) [inline helm/seekret/templates/seekret.yaml:183]: revert

**Review by asdftamir — CHANGES_REQUESTED** [review]

File: helm/README.md [hunk 2]
```diff
@@ -10,13 +10,17 @@ Install this chart using:
 
 ```bash
  cd helm/seekret
- helm install seekret --namespace seekret --create-namespace .
+ helm install seekret .
 ```
 
+We create a namespace called seekret-injector. This is hard coded since the certificates are signed on those domains.
```
Comment (asdftamir) [inline helm/README.md:16]: does this work if the namesapce already exists?

**Review by asdftamir — CHANGES_REQUESTED** [review]

File: helm/seekret/templates/seekret.yaml (hunk 1 above)
Comment (guyarb) [reply helm/seekret/templates/seekret.yaml]: done

File: helm/README.md (hunk 2 above)
Comment (guyarb) [reply helm/README.md:16]: should not exist. this is a unique namespace, if someone is using the same namespace it is weird
if we are already installed - you'll need to delete-install or to upgrade

File: tests/test_helm.sh [hunk 3]
```diff
@@ -53,16 +52,17 @@ ACCESS_KEY=$1
 SECRET_ACCESS_KEY=$2
 BUCKET_NAME=$3
 echo "Creating seekret's helm sniffer setup"
-run helm install seekret --namespace $NAMESPACE --create-namespace ${CURRENT_DIR}/../helm/seekret/ --set s3.accessKey=${ACCESS_KEY} --set s3.secretKey=${SECRET_ACCESS_KEY} --set s3.bucketName=${BUCKET_NAME}
+run helm install seekret ${CURRENT_DIR}/../helm/seekret/ --set s3.accessKey=${ACCESS_KEY} --set s3.secretKey=${SECRET_ACCESS_KEY} --set s3.bucketName=${BUCKET_NAME}
 cleanupOnError $?
-run kubectl wait --for=condition=available --timeout=60s deployment/seekret-sidecar-injector -n kube-system
+run kubectl wait --for=condition=available --timeout=60s deployment/seekret-sidecar-injector -n seekret-injector
```
Comment (asdftamir) [inline tests/test_helm.sh:57]: $NAMESPACE?

**Review by asdftamir — CHANGES_REQUESTED** [review]

**Review by asdftamir — APPROVED** [review]

**Pull request merged**